source/content/posts/CoreData.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "CoreData"
+date: 2018-01-10T15:06:30Z
+draft: false
+---
+
+# Usando CoreData por primera vez
+
+Hasta hace algunos días no había tenido la necesidad de utilizar un manejador de base de datos en las aplicaciones que he desarrollado, para manejar la persistencia de datos utilizaba los [UserDefaults](https://developer.apple.com/documentation/foundation/userdefaults).
+<p align="justify">
+Pero me ví con la necesidad de almacener una cantidad de datos mucho mayor, podría utilizar los **UserDefaults** pero no es óptimo, los mismos desarrolladores de **Apple** recomiendan implementar los UserDefaults para almacenar información puntual acerca del usuario y sus preferencias, por lo que necesitaba utilizar un manejador de base de datos y esto era algo nuevo para mí, tenía que decidir entre las diferentes bases de datos compatibles con dispositivos iOS y una entre ellas se encontraba **CoreData** que es un framework desarrollada y recomendada por **Apple**.</p>
```

Comment:
I think you have to change "Pero me ví con la necesidad " to "Pero me ví en la necesidad"

Reply:
si, suena mejor con esa frase